invoices_test.go

```diff
@@ -815,6 +815,90 @@ func TestInvoices_Create(t *testing.T) {
 				<a name="mark_failed" href="https://your-subdomain.recurly.com/v2/invoices/1016/mark_failed" method="put"/>
 			</charge_invoice>
 			<credit_invoices type="array">
+<credit_invoice>
```

Comment:
Minor nit but let's adjust indentation in this entire block (including below). These are large XML blocks so it's easier to debug if they're indented consistently.